addons/website_sale_options/views/website_sale_options_templates.xml

```diff
@@ -63,7 +63,7 @@
                    }'/>
                   <span class="oe_price" style="white-space: nowrap;"
                     t-att-data-product-id="product.id"
-                    t-field="product.with_context(quantity=1).website_price"
+                    t-esc="product.website_price / quantity"
```

Comment:
First, we need this price to be computed in function of the quantity (for pricelist compliance)
Second website_price is the computed price of the (soon to be) sale order line.
and we want the unit price.
https://github.com/odoo/odoo/blob/10.0/addons/website_sale/models/product.py#L202